### PR TITLE
gateway: br-localnet really should be br-local

### PR DESCRIPTION
**Pull request by girishmg**
`localnet` is a very well known concept in OVN, so br-localnet gives
a wrong idea to someone who is starting new with ovn-kubernetes. So,
br-localnet should be renamed to br-local (it is similar to what
br-local does in OpenStack).

Signed-off-by: Girish Moodalbail <gmoodalbail@nvidia.com>

### CODE REVIEW

File: vagrant/provisioning/vm_config.conf.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@ k8smaster:
...
   host_name: "k8smaster.ovn"
   public-ip: "10.10.0.11"
-  memory: 2048
+  memory: 3192
   cpus: 2
 k8sminion1:
   short_name: "k8sminion1"
   host_name: "k8sminion1.ovn"
   public-ip: "10.10.0.12"
-  memory: 2048
```
Comment (shettyg) [inline vagrant/provisioning/vm_config.conf.yml]: Unrelated change?

Comment (girishmg) [reply vagrant/provisioning/vm_config.conf.yml]: Yeah, will revert it. BTW, Vagrant VM's 2048 memory isn't sufficient for daemonset mode. Some of the PODs don't start and remain in pending with 'insufficient memory' error.

**Comment by girishmg** [comment]
@shettyg PTAL. Thanks
